runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java

```diff
@@ -110,7 +119,7 @@ private void renderPlayerOverlay(Graphics2D graphics, Player actor, Color color)
 
 		if (config.showClanRanks() && actor.isClanMember())
 		{
-			final ClanMemberRank rank = clanManager.getRank(name);
+			final ClanMemberRank rank = clanManager.getRank(Text.sanitize(actor.getName()));
```

Comment:
You should reuse this. E.g create variable sanitizedName, and use it in both if/else branches and here